schemas/context/action.schema.json

```
@@ -2,11 +2,20 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "$id": "https://fdc3.finos.org/schemas/next/context/action.schema.json",
   "title": "Action",
-  "description": "A representation of an FDC3 Action (specified via a Context or Context & Intent) that can be inserted inside another object, for example a chat message.\n\nThe action may be completed by calling `fdc3.raiseIntent()` with the specified Intent and Context, or, if only a context is specified, by calling `fdc3.raiseIntentForContext()` (which the Desktop Agent will resolve by presenting the user with a list of available Intents for the Context).\n\nAccepts an optional `app` parameter in order to specify a specific app.",
+  "description": "A representation of an FDC3 Action (specified via a Context or Context & Intent) that can be inserted inside another object, for example a chat message.\n\nThe action may be completed by calling:\n- `fdc3.raiseIntent()` with the specified Intent and Context\n- `fdc3.raiseIntentForContext()` if only a context is specified, (which the Desktop Agent will resolve by presenting the user with a list of available Intents for the Context).\n- `channel.broadcast()` with the specified Context, if the `broadcast` action has been defined.\n\nAccepts an optional `app` parameter in order to specify a specific app.",
   "allOf": [{
       "type": "object",
       "properties": {
         "type": { "const": "fdc3.action" },
+        "action": {
+          "title": "Action Type",
+          "description": "The **action** field indicates the type of action:\n- **raiseIntent** :  If no action or `raiseIntent` is specified, then the current default behaviour will be used with `fdc3.raiseIntent` / `fdc3.raiseIntentForContext`\n- **broadcast** : If `broadcast` action is specified, it will call `fdc3.getOrCreateChannel(channelId)`to retrieve the channel, then broadcast the context to that channel with `channel.broadcast(context)`. If no, channel has been specified, it will try to broadcast to the current channel (`fdc3.getCurrentChannel()`)",
```

```suggestion
          "description": "The **action** field indicates the type of action:\n- **raiseIntent** :  If no action or `raiseIntent` is specified, then the current default behaviour will be used with `fdc3.raiseIntent` / `fdc3.raiseIntentForContext`\n- **broadcast** : If `broadcast` action is specified, it will call `fdc3.getOrCreateChannel(channelId)`to retrieve the channel, then broadcast the context to that channel with `channel.broadcast(context)`. If no channel has been specified, it will try to broadcast to the current channel (`fdc3.getCurrentChannel()`)",
```

          'If no channel has been specified'? or 'If no (broadcast action is specified), and a channel has been specified'? I feel like the comma introduces some ambiguity.

---

@symphony-jean-michael I think you just need to merge the suggestion above from Hugh, or rework that sentence and then this is good to merge.